neo4j/content.md

```diff
@@ -15,11 +15,11 @@ docker run \
     %%IMAGE%%
 ```
 
-which allows you to access neo4j through your browser at [http://localhost:7474](http://localhost:7474).
-
 This binds two ports (`7474` and `7687`) for HTTP and Bolt access to the Neo4j API. A volume is bound to `/data` to allow the database to be persisted outside the container.
 
-By default, this requires you to login with `neo4j/neo4j` and change the password. You can, for development purposes, disable authentication by passing `--env=NEO4J_AUTH=none` to docker run.
+Once running, you can use the Neo4j Aura console which includes graph tools for visualizations, data exploration, and monitoring for free. No subscription is required. Simply create a self-managed instance and connect to http://localhost:7474. Alternatively, you can use the Neo4j Browser, a web-based user interface for interacting with Neo4j that is included with the Neo4j installation. To access the Neo4j Browser, open a web browser and navigate to http://localhost:7474.
```

Comment:
@jennyowen 

Once running, you can use the [Neo4j Aura console](https://console-preview.neo4j.io/self-managed) which includes graph tools for visualizations, data exploration, and monitoring for free. No subscription is required. Simply create a self-managed instance and connect to `http://localhost:7474`.

Alternatively, you can use the Neo4j Browser, a web-based user interface for interacting with Neo4j that is included with the Neo4j installation. To access the Neo4j Browser, open a web browser and navigate to [http://localhost:7474](http://localhost:7474).